eth/downloader/queue.go

```diff
@@ -39,9 +39,10 @@ const (
 )
 
 var (
-	blockCacheItems      = 8192             // Maximum number of blocks to cache before throttling the download
-	blockCacheMemory     = 64 * 1024 * 1024 // Maximum amount of memory to use for block caching
-	blockCacheSizeWeight = 0.1              // Multiplier to approximate the average block size based on past ones
+	blockCacheMaxItems     = 8192             // Maximum number of blocks to cache before throttling the download
+	blockCacheInitialItems = uint64(2048)     // Initial number of blocks to start fetching, before we know the sizes of the blocks
```

Comment:
seems weird to type it, why not leave as untyped constant?

Reply:
Yup, fixed